vividus-extension-visual-testing/src/main/java/org/vividus/visual/screenshot/AdjustingCoordsProvider.java

```diff
@@ -17,32 +17,29 @@
 package org.vividus.visual.screenshot;
 
 import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.SearchContext;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.vividus.selenium.manager.IWebDriverManager;
+import org.vividus.selenium.screenshot.AbstractAdjustingCoordsProvider;
 import org.vividus.selenium.screenshot.IScrollbarHandler;
 import org.vividus.ui.context.IUiContext;
 
 import ru.yandex.qatools.ashot.coordinates.Coords;
-import ru.yandex.qatools.ashot.coordinates.WebDriverCoordsProvider;
 
-public class AdjustingCoordsProvider extends WebDriverCoordsProvider
+public class AdjustingCoordsProvider extends AbstractAdjustingCoordsProvider
```

Comment:
```suggestion
public class WebAdjustingCoordsProvider extends AbstractAdjustingCoordsProvider
```
?
and about moving it to `vividus-plugin-web-app`?